resources/lang/uk/model_validation/fulfillments.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ *    Copyright (c) ppy Pty Ltd <contact@ppy.sh>.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'username_change' => [
+        'only_one' => '',
+        'insufficient_paid' => '',
+        'reverting_username_mismatch' => '',
+    ],
+    'supporter_tag' => [
+        'insufficient_paid' => 'Сума платежу недостатня для придбання osu!supporter (: actual > :expected)',
```

Comment:
There shouldn't be a space in `:actual` but I also think this line isn't supposed to be translated.
I've already fixed it on crowdin.

Reply:
Have pulled down updates 👍